lib/config_generator.js

```diff
@@ -21,6 +21,11 @@ exports.generate = function(config) {
   config.stubStatus = config.stubStatus || false;
   config.websocketProxy = config.websocketProxy || {};
 
+  if ((config.tlsCertificateFile && !config.tlsCertificateKeyFile) ||
+       (!config.tlsCertificateFile && config.tlsCertificateKeyFile)) {
+    throw new Error('Certificate file and key file must both be set!');
```

Comment:
Could you name the configuration options, i.e. `tlsCertificateFile ` instead of `certificate file`. Also, the error should describe the ability to specify neither option to force proxrox